test/e2e/common/util_cleanup.go

```diff
@@ -333,28 +362,38 @@ func CleanUp() bool {
 		"pvcCount", pvcCount,
 		"pvCount", pvCount,
 		"msvCount", msvCount,
-		"delPodsErr", delPodsErr,
-		"delPvcErr", delPvcErr,
-		"delPvErr", delPvErr,
-		"delMsvErr", delMsvErr,
+		"err", errs,
 		"poolFinalizerDeleted", poolFinalizerDeleted,
 		"delPoolFinalizeErr", delPoolFinalizeErr,
 	)
 
-	scList, delScErr := gTestEnv.KubeInt.StorageV1().StorageClasses().List(context.TODO(), metav1.ListOptions{})
-	if delScErr == nil {
+	scList, err := gTestEnv.KubeInt.StorageV1().StorageClasses().List(context.TODO(), metav1.ListOptions{})
+	if err == nil {
 		for _, sc := range scList.Items {
 			if sc.Provisioner == "io.openebs.csi-mayastor" && sc.Name != "mayastor-iscsi" && sc.Name != "mayastor-nvmf" {
 				logf.Log.Info("Deleting", "storageClass", sc.Name)
 				_ = gTestEnv.KubeInt.StorageV1().StorageClasses().Delete(context.TODO(), sc.Name, metav1.DeleteOptions{GracePeriodSeconds: &ZeroInt64})
 			}
 		}
 	} else {
-		logf.Log.Info("Error listing storage classes", "error", delScErr)
+		errs = append(errs, err)
+	}
+
+	//tidy up namespaces?
```

Comment:
Should this comment be removed? You are removing the namespaces.

Reply:
Done.